### PR TITLE
Docs: license calls in tutorials and Framework.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: license calls in tutorials and Framework.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/framework/custom-editor-creator.md
Comment (gorzelinski) [inline docs/framework/custom-editor-creator.md:1]: This change may not be for now, but we should probably move some guides (like this one) to corresponding folders. It took me a sec to find it.

**Review by gorzelinski — APPROVED** [review]
Looks good 👍

File: docs/framework/contributing/testing-environment.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ import { ClassicEditor, Essentials, Paragraph } from 'ckeditor5';
 
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/framework/contributing/testing-environment.md:171]: We do intend to have a testing key, so I just placeholdered it, actually, as we have nothing else, now.

File: docs/framework/custom-editor-creator.md
Comment (godai78) [reply docs/framework/custom-editor-creator.md:1]: LEt's have a follow-up for that.

**Review by gorzelinski — APPROVED** [review]

**Pull request merged**